types/bootstrap/js/dist/base-class.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Base Component
+ *
+ * @link https://github.com/twbs/bootstrap/blob/main/js/src/base-component.js
+ */
+export class BaseClass {
```

Comment:
```suggestion
export default class BaseClass {
```
sorry to be pita, why it cannot be default? To match the source as close as possible?
this would impact added imports:
```ts
import BaseComponent from './base-component'
```

Reply:
I renamed the component. Thanks